src/test/TEST_ij/amgdd.saved.florentia

```diff
@@ -1,64 +1,64 @@
 # Output file: amgdd.out.900
-BoomerAMG-DD Iterations = 10
-Final Relative Residual Norm = 1.928307e-09
+BoomerAMG-DD Iterations = 9
```

Comment:
Some of these differences are pretty substantial.  Do you have an explanation for them?

Reply:
Most of the saved outputs are quite old. There have been several new parts of the code ported to the GPU (for the sycl backend), several bug fixes, and new oneAPI releases since the previous outputs were saved. I would not put much stock in the old saved outputs. I checked the new outputs against what we get with CPU and cuda builds, and the numbers look reasonable to me, so hopefully the new .saved files should be a good baseline going forward.